metrics.nim

```diff
@@ -331,7 +332,8 @@ when defined(metrics):
   proc validateCounterLabelValues(counter: Counter, labelValues: LabelsParam): Labels {.raises: [Defect, ValueError].} =
     result = validateLabelValues(counter, labelValues)
```

Comment:
the lookup here also needs to be behind the lock in case two threads are updating the metric at the same time

Reply:
There's no lookup in `validateLabelValues()`: https://github.com/status-im/nim-metrics/blob/16ec7aeccc2020666d6b7c63288153f3917222fb/metrics.nim#L199-L209

Reply:
`if result notin counter.metrics:`

Reply:
Right.